### PR TITLE
Newsletter placed inside left column

### LINKED ISSUE #9907: [BOOM-3749] Problème d'affichage sur module natif greffé
> This issue has been migrated from the Forge. Read the [original ticket here](http://forge.prestashop.com/browse/BOOM-3749).

- _**Reporter:**_ bernon
- _**Created at:**_ Thu, 31 Aug 2017 18:09:16 +0200

<p>Création d'une colonne gauche sur la page index.<br/>
Module Newsletter greffé sur la colonne de Gauche et remonté en première position.</p>
        
* How to reproduce the issue ?

N/A

### PR DESCRIPTION
**Pull request by YeLnatSs**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Newsletter placed inside left column
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | Does it break backward compatibility? no
| Deprecations? | Does it deprecate an existing feature? no
| Fixed ticket? | Fixes #9907
| How to test?  | Add newsletter module inside, left column in BO positions page.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/13619)

### CODE REVIEW

File: themes/classic/_dev/css/partials/_variables.scss [hunk 1]
```diff
@@ -3,6 +3,7 @@ $gray-darker: #232323;
 $gray: #7A7A7A;
 $gray-light: #F1F1F1;
 $gray-light-second: #DBDBDB;
+$gray-light-third: #eee;
```
Comment (Quetzacoalt91) [inline themes/classic/_dev/css/partials/_variables.scss]: I'm unsure we need to a a variable for this task.

**Review by Quetzacoalt91 — CHANGES_REQUESTED** [review]
This solution is unfortunately not generic enough in my opinion. As soon as we get out the perimeter of the described issue, we still encounter the visual anomaly.

Capture d’écran du 2019-05-01 09-25-53

File: themes/classic/_dev/css/partials/_variables.scss (hunk 1 above)
Comment (YeLnatSs) [reply themes/classic/_dev/css/partials/_variables.scss]: No problem,I will take it off.

**Comment by YeLnatSs** [comment]
> This solution is unfortunately not generic enough in my opinion. As soon as we get out the perimeter of the described issue, we still encounter the visual anomaly.
> 
> Capture d’écran du 2019-05-01 09-25-53

I will check it again

File: themes/classic/_dev/css/components/categories.scss [hunk 2]
```diff
@@ -344,6 +344,33 @@
   }
 }
 
+#left-column,
+#right-column {
```
Comment (Quetzacoalt91) [inline themes/classic/_dev/css/components/categories.scss:348]: Okay, you handle both columns, but what if we hook the module in another place? Have you checked the responsiveness of the module in all different part of the page?

**Comment by marionf** [comment]
This PR has been replaced by: https://github.com/PrestaShop/PrestaShop/pull/14937

**Pull request closed without merging**